ci-operator/step-registry/rhoai/patch-component-deployment/odh-model-controller/rhoai-patch-component-deployment-odh-model-controller-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: rhoai-patch-component-deployment-odh-model-controller
+  from_image:
+    namespace: ocp
+    name: "4.17"
```

Comment:
I guess we are using OCP 4.18 for all other tests.

Reply:
This is just to get `oc` to patch the resource, it is not for the actual cluster version.
Actual cluster is installed in `hypershift-hostedcluster-create` which uses latest HCP